### ISSUE TITLE
 Navigation slider is missing aria-label

### ISSUE THREAD

**Issue by ronnymikalsen** [NationalLibraryOfNorway/ngx-mime#260]
**Describe the bug**
 Navigation slider should have a meaningful aria-label

**To Reproduce**

1.  Go to 'http://localhost.nb.no:4200/demo'
2.  Open dashboard mode
4.  Analyze page with Axe Chrome plugin

**Expected behavior**
Should pass all checks

**Which versions of NGX-Mime, Angular, Material, OS, TypeScript, browsers are affected?**
All

**Comment by ronnymikalsen** [comment]
Fixed by #268
